common/api-review/util.api.md

```diff
@@ -329,6 +329,11 @@ export const isValidFormat: (token: string) => boolean;
 // @public
 export const isValidTimestamp: (token: string) => boolean;
 
+// Warning: (ae-missing-release-tag) "isWebWorker" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export function isWebWorker(): boolean;
+
```

Comment:
Apparently I missed the doc generate step on the user-created PR that added this function (#8315). My bad.